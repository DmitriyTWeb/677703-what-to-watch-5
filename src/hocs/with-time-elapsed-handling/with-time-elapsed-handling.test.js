import React from "react";
import renderer from "react-test-renderer";
import withTimeElapsedHandling from "./with-time-elapsed-handling";

const MockComponent = () => {
  return (
    <div>
    </div>
  );
};

const MockComponentWrapped = withTimeElapsedHandling(MockComponent);

it(`withTimeElapsedHandling is rendered correctly`, () => {
  const tree = renderer.create(
      <MockComponentWrapped
        videoRef={{}}
        isPlaying={false}
      />
  );

  expect(tree).toMatchSnapshot();
});

