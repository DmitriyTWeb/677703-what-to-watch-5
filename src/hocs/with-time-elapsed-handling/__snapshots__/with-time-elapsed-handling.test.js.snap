// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withTimeElapsedHandling is rendered correctly 1`] = `<div />`;
